docs/reference/ccr/bi-directional-disaster-recovery.asciidoc

```diff
@@ -0,0 +1,214 @@
+[role="xpack"]
+[[ccr-disaster-recovery-bi-directional-tutorial]]
+=== Tutorial: Disaster recovery based on bi-directional {ccr}
+++++
+<titleabbrev>Bi-directional disaster recovery</titleabbrev>
+++++
+
+Learn how to set up disaster recovery between two clusters based on
+bi-directional {ccr}. The following tutorial is designed for immutable indices
+(data no longer need to be updated after writing) like logs, metrics, observability, 
+and security events. If you have mutable indices, you should follow <<disaster-recovery,the uni-directional cross-cluster replication disaster recovery tutorial>>.
```

Comment:
Turns out data stream supports `update_by_query` and `delete_by_query`. https://www.elastic.co/guide/en/elasticsearch/reference/master/use-a-data-stream.html#update-docs-in-a-data-stream-by-query. As such, I will remove the "immutable" restriction. 
 
```suggestion
bi-directional {ccr}. The following tutorial is designed for data streams which supports <<{ref}/use-a-data-stream.html#update-docs-in-a-data-stream-by-query,update_by_query>> and <<{ref}/use-a-data-stream.html#delete-docs-in-a-data-stream-by-query,delete_by_query>>. You can only perform these actions on the leader index. 
```